dojo/tools/qualys_webapp/parser.py

```diff
@@ -28,11 +35,17 @@ def get_cwe(cwe):
 # Inputs are a list of endpoints and request/response pairs and doctors
 # them to fit their respective model structures and the adds them to a
 # newly generated Finding
-def attach_extras(endpoints, requests, responses, finding):
+def attach_extras(endpoints, requests, responses, finding, date, qid):
     if finding is None:
         finding = Finding()
         finding.unsaved_req_resp = list()
         finding.unsaved_endpoints = list()
+        finding.date = date
+        finding.unique_id_from_tool = str(qid)
```

Comment:
The qid is not unique in qualys because in can be found on multiple targets. This is not the idea of unique_id_from_tool. But you can add it in the description.

Reply:
Would you consider it reasonable to add another database column for something like "vuln_id_from_tool"? We'd really like this information to be available for de-duplication and reporting purposes.

Reply:
I don't know if many tools have there own ids? If yes, it is a great idea.

Reply:
I know at least Veracode does.